docs/framework/react/guides/query-options.md

```diff
@@ -32,3 +32,16 @@ queryClient.setQueryData(groupOptions(42).queryKey, newGroups)
 [//]: # 'Example1'
 
 For Infinite Queries, a separate [`infiniteQueryOptions`](../reference/infiniteQueryOptions.md) helper is available.
+
+You can still override some options at the component level. A very common and useful pattern is to create per-component [`select`](../guides/render-optimizations.md#select) functions:
+
```

Comment:
```suggestion

[//]: # 'Example2'

```

Reply:
Remind me, these markings are hidden headings/hash links? I did put an "Example2" notation down a bit further.

Reply:
They mark the beginning and end of Example 2, so that other framework adapters can potentially overwrite the examples with their own implementations, e.g. from the vue docs:

https://github.com/TanStack/query/blob/9739c8d12cca44a8671cad5735af9412bd815532/docs/framework/vue/guides/background-fetching-indicators.md#L1